app/build.gradle

```diff
@@ -40,7 +40,7 @@ android {
   }
   splits {
     // See: https://developer.android.com/studio/build/configure-apk-splits
-    density {
+    abi {
```

Comment:
So I don't think we can actually approach it this way since this seems to be disabling density-based splitting entirely (which isn't what we want).

Reply:
Okay, Ben. Got it.